.changeset/dull-cheetahs-applaud.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/components": patch
+---
+
+Migrate `Flash` to TypeScript
```

Comment:
Should this changeset mention that `scheme`  was removed?

Reply:
`scheme` actually wasn't being used. So removing it didn't change anything.